src/app/pages/home-page/offer-item-page/offer-item-page.component.html

```diff
@@ -1,38 +1,37 @@
-<div *ngIf="offer$ | async as offer" class="item">
+<div *ngIf="(isLoading$ | async) === false" class="item">
   <app-offer-item [offer]="offer"></app-offer-item>
 
   <mat-tab-group mat-stretch-tabs class="tab">
-    <mat-tab label="description">
+    <mat-tab label="{{ 'CARD.DESCRIPTION' | translate }}">
       <div class="item__header">
-        <div class="item__date-end">valid to {{ offer.dateEnd }}</div>
+        <div class="item__date-end">
+          {{ 'CARD.VALID_TO' | translate }} {{ offer.dateEnd }}
+        </div>
         <div class="item__actions">
           <button
             *ngIf="offer.promocode"
             mat-button
-            class="button"
+            class="button button_promocode"
```

Comment:
I recommend to extract button styles to shared button and add some modificators.

Reply:
didn't have time then, did now